Data/market-hours/market-hours-database.json

```diff
@@ -3671,8 +3671,14 @@
       ],
       "saturday": [],
       "holidays": [
+        "1/2/2017",
```

Comment:
Please project forward for these holiday dates. What are the dates 2018, 2019, 2020